pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -736,15 +741,27 @@ func (l *listUsersQuery) expandExclusion(
 
 	var baseError error
 	go func() {
-		resp := l.expandRewrite(ctx, req, rewrite.Difference.GetBase(), baseFoundUsersCh)
```

Comment:
`expandRewrite` will panic in certain circumstances (See https://github.com/openfga/openfga/blob/main/pkg/server/commands/listusers/list_users_rpc.go#L420). Have you tried to trigger a panic on this line, rather than using dependency injection?